pyanaconda/anaconda.py

```diff
@@ -31,9 +31,11 @@
 from pyanaconda import addons
 
 import logging
-log = logging.getLogger("anaconda")
 stdoutLog = logging.getLogger("anaconda.stdout")
 
+from pyanaconda.ilog import get_logger
```

Comment:
Why is the submodule called `ilog`? I know we had problems with name collisions between anaconda submodules and python modules, but that should be fixed now. Same for `ihelp`, `iutil` and `isignal`. Is there another reason for these names?